### PR TITLE
Addresses #122, setpoint diversity.

### PR DESCRIPTION
**Pull request by joseph-robertson**

### CODE REVIEW

File: data/other/input.py [hunk 1]
```diff
@@ -6,11 +6,135 @@
...
+    
+    base_temp_c = base_temp_f / 1.8
+    if is_heating:
+        deg_days = sum([(base_temp_c-x) for x in daily_dbs if x < base_temp_c])
+    else:
+        deg_days = sum([(x-base_temp_c) for x in daily_dbs if x > base_temp_c])
+    return deg_days * 1.8
+    
+def _get_daily_dbs(epwfile):
+
```
Comment (ejhw) [inline data/other/input.py]: add docstring, e.g., dbs=drybulbs?

File: resources/options_lookup.tsv [hunk 2]
```diff
@@ -869,6 +869,10 @@ Heating Setpoint	73F	ResidentialHVACHeatingSetpoints	weekday_setpoint=73	weekend
...
+Heating Setpoint	62F	ResidentialHVACHeatingSetpoints	weekday_setpoint=62	weekend_setpoint=62	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
+Heating Setpoint	64F	ResidentialHVACHeatingSetpoints	weekday_setpoint=64	weekend_setpoint=64	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
+Heating Setpoint	66F	ResidentialHVACHeatingSetpoints	weekday_setpoint=66	weekend_setpoint=66	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
+Heating Setpoint	67F	ResidentialHVACHeatingSetpoints	weekday_setpoint=67	weekend_setpoint=67	use_auto_season=false	season_start_month=Jan	season_end_month=Dec																																							
 Cooling Setpoint	FIXME Location Average	ResidentialHVACCoolingSetpoints	weekday_setpoint=76	weekend_setpoint=76	use_auto_season=false	season_start_month=Jan	season_end_month=Dec
```
Comment (shorowit) [inline resources/options_lookup.tsv]: Can we now remove "FIXME Location Average" options from the Heating Setpoint and Cooling Setpoint parameters?

File: project_resstock_comed/housing_characteristics/Cooling Setpoint.tsv [hunk 3]
```diff
@@ -1,2 +1,217 @@
-Option=69F	Option=74F	Option=76F	Option=78F	Option=FIXME Location Average
-0	0	0	0	1
+Dependency=Location EPW	Option=80F	Option=81F	Option=82F	Option=83F	Option=84F	Option=85F	Option=86F	Option=87F	Option=88F	Option=89F	Option=90F	Option=92F	Option=93F	Option=94F	Option=95F	Option=96F	Option=97F
```
Comment (shorowit) [inline project_resstock_comed/housing_characteristics/Cooling Setpoint.tsv]: @ejhw Cooling setpoints up to 97F???

Comment (ejhw) [reply project_resstock_comed/housing_characteristics/Cooling Setpoint.tsv]: Yeah we noticed. Joe's looking into it.

File: resources/options_lookup.tsv (hunk 2 above)
Comment (joseph-robertson) [reply resources/options_lookup.tsv]: @shorowit, I believe we could? @ejhw?

Comment (shorowit) [reply resources/options_lookup.tsv]: If no projects are using it anymore, then it can be removed.

**Pull request merged**